packages/firebase_admob/android/src/main/java/io/flutter/plugins/firebaseadmob/FirebaseAdMobPlugin.java

```diff
@@ -24,6 +27,12 @@
 
   RewardedVideoAdWrapper rewardedWrapper;
 
+  static Function<UnifiedNativeAd, View> nativeAdGenerator;
```

Comment:
nit: I'd consider calling this factory

Reply:
done https://github.com/FirebaseExtended/flutterfire/pull/169/files#diff-7c229379ed87228a4d21cd88176456f7R30